### ISSUE TITLE
🚀 [Feature Request] - Adding images to the blog cards. [GSSoC'23]

### ISSUE THREAD

**Issue by 1Shubham7** [amupedia2021/Project-Amupedia#377]
### Is your feature request related to a problem ? Please describe. ✍️

Enhancement:
In the blog section, There are blog cards, with same image. I can add images to those cards making them more interesting and fun.

Screenshot:
![imagecard](https://github.com/amupedia2021/Project-Amupedia/assets/116020663/c0de5568-9428-4481-896f-dac146da3b59)

### Describe briefly how you would implement this enhancement.

```bash
I will find some images, edit them and then go to the code base and add images to it.
```

### Additional context or screenshots

Additional context : Please assign me this issue under GSSoC'23

### Are you contributing for a program?

GSSoC2023

### Code of Conduct

- [X] I have read and agree to the Contributing Guidelines of this project.

**Comment by 1Shubham7** [comment]
Please assign me this issue under GSSoC'23

**Comment by Abhinavcode13** [comment]
@anmode check this once !

**Comment by Abhinavcode13** [comment]
Start working as of now @1Shubham7

**Comment by 1Shubham7** [comment]
Thank you @Abhinavcode13 . I will surely not disappoint you.

**Comment by tayyab-ilyas** [comment]
This issue has been resolved #484
